documentation/markup.rst

```diff
@@ -678,8 +676,12 @@ Roles
 As said before, Sphinx uses interpreted text roles to insert semantic markup in
 documents.
 
-Names of local variables, such as function/method arguments, are an exception,
-they should be marked simply with ``*var*``.
+There are a couple of cases where roles can be omitted in favor of
+simpler markup:
+
+* Function and method arguments can be marked with ``*arg*``.
+* ``True``, ``False``, ``None``, and variables can be marked with
+  :literal:`\`\`...```.
 
 For all other roles, you have to write ``:rolename:`content```.
```

Comment:
```suggestion
```

I've moved the meaningful content of this (the syntax example) up above where it fits better, and I suggest just eliding this; it duplicates the previous linked section that discusses the general reST/Sphinx syntax, whereas this one describes the CPython-specific semantics, and its not "all" other roles (there are others mentioned in the aforementioned section).